EasyPost/Services/AddressService.cs

```diff
@@ -2,7 +2,6 @@
 using System.Threading;
 using System.Threading.Tasks;
 using EasyPost._base;
-using EasyPost.BetaFeatures.Parameters;
```

Comment:
issue: in the future, handle removing imports as a separate PR since the PRs purpose is to update docstrings